specifyweb/businessrules/agent_rules.py

```diff
@@ -6,20 +6,41 @@
 def agent_delete_blocked_by_related_specifyuser(agent):
     try:
         Specifyuser.objects.get(agents=agent)
+        user = Specifyuser.objects.get(agents=agent)
     except Specifyuser.DoesNotExist:
         return
-    raise BusinessRuleException("agent cannot be deleted while associated with a specifyuser")
+    raise BusinessRuleException(
```

Comment:
(optional) Is there a way to get rid of this business rule in favor of adding this information to the 
response from `/api/delete_blockers/`?

Since front-end already has a nice UI for delete blockers, doing it though that will save us some work

Reply:
Added Agent.specifyuser to deletion_blockers in 56cb36f.